rpc/walletrpc/methods_test.go

```diff
@@ -40,7 +40,7 @@ func TestGetSyncHeight(t *testing.T) {
 		log.Error("openWallet fail", "err", err.Error())
 		return
 	}
-	fmt.Printf("TestGetSyncHeight: %v\n", w.Manager.SyncedTo().Height)
+	fmt.Printf("TestGetSyncHeight: %v\n", w.Manager.SyncedTo().Order)
```

Comment:
SyncHeight  => SyncOrder

Reply:
The wallet uses "height" for "order," which is just a replacement for the meaning of the word.